### DISCUSSION TITLE
C++ header files for Windows

### DISCUSSION THREAD

**Discussion by RubyAI132** [Yaskawa-Global/YMConnect#3] [Q&A] [answered]
Hello I want to follow the quick start tutorial
I can't find the C++ header files for Windows

Is there a download source

**Comment by ted-miller** [comment]
Hi @RubyAI132,
The header file is included in the zip on the [Releases page](https://github.com/Yaskawa-Global/YMConnect/releases).

**Reply by ted-miller** [reply, marked as answer]
Be sure to expand the Assets section to see the release downloads.

![image](https://github.com/Yaskawa-Global/YMConnect/assets/4623811/8e5e4bab-6045-41b7-a34b-9355d6b2d16a)

**Comment by RubyAI132** [comment]
I see, thank you
